src/language/__tests__/lexer-test.js

```diff
@@ -129,7 +131,7 @@ describe('Lexer', () => {
   it('updates line numbers in error for file context', () => {
     let caughtError;
     try {
-      const str = '' + '\n' + '\n' + '     ?\n' + '\n';
+      const str = ['', '', '     ?', ''].join('\n');
```

Comment:
It was triggering `no-useless-concat` rule.
@mjmahone If you think it became less readable I can revert it back and just add ESLint comment that will disable `no-useless-concat` rule for this line.

Reply:
No I think this is as readable. Both ways are mostly nonsense.